momentum_scanner_intraday.py

```diff
@@ -102,6 +110,13 @@ def find_rets_24h(vols):
     rets24h.index.name = "Token name"
     return rets24h
 
+def add_volume_marketcap(df):
+    for i in df.index:
```

Comment:
Use `i` to mean integers, and label columns. I _think_ this should say `for token in df["tokens"]`, and that should be threaded through to work, but I can't tell as written.

Reply:
changed